### PR TITLE
feat(underpriced-message): handle tx underpriced error

### PR DESCRIPTION
**Pull request by envin3**

### CODE REVIEW

File: src/store/swap/utils/pegin-with-deposit-address.js
Comment (oliviera9) [inline src/store/swap/utils/pegin-with-deposit-address.js:1]: It's impossible for a peg-in with deposit to have this problem!

File: src/store/swap/utils/pegin-with-wallet.js [hunk 1]
```diff
@@ -27,7 +28,9 @@ const peginWithWallet = async ({ swap, ptokenFrom, ptokenTo, dispatch }) => {
       dispatch(
         updateInfoModal({
           show: true,
-          text: 'Error during pegin, try again!',
+          text: _err.message.includes('underpriced')
+            ? "You transaction wasn't accepted by the network as underpriced. Please try again increasing the gasprice from your wallet before signing."
+            : 'Error during pegin, try again!',
           showMoreText: _err.message ? _err.message : _err,
```
Comment (oliviera9) [inline src/store/swap/utils/pegin-with-wallet.js:34]: I suspect here we are still showing the entire long trace.

Comment (envin3) [reply src/store/swap/utils/pegin-with-wallet.js:34]: Yes, as discussed.

File: src/store/swap/utils/pegin-with-deposit-address.js
Comment (envin3) [reply src/store/swap/utils/pegin-with-deposit-address.js:1]: Removed

File: src/store/swap/utils/pegin-with-wallet.js (hunk 1 above)
Comment (oliviera9) [reply src/store/swap/utils/pegin-with-wallet.js:34]: So it's desirable behavior to have the long trace when opening the details, right?

Comment (oliviera9) [reply src/store/swap/utils/pegin-with-wallet.js:34]: in this case error.message already contains the stack trace

**Review by oliviera9 — APPROVED** [review]
LGTM. If you have time consider putting the new messages in a constant you reuse.

**Pull request merged**